lib/vdsm/storage/lvm.py

```diff
@@ -887,15 +887,19 @@ def getVg(self, vgName):
         return vg
 
     def getVgs(self, vgNames):
-        """Reloads all the VGs of the set.
+        """Reload all the VGs of the set.
 
         Can block for suspended devices.
-        Fills the cache but not uses it.
-        Only returns found VGs.
+        Fill the cache but do not use it.
+        Return only found VGs.
         """
         self.stats.miss()
-        return [vg for vgName, vg in six.iteritems(self._reloadvgs(vgNames))
-                if vgName in vgNames]
+        try:
+            return [vg for vgName, vg in six.iteritems(
+                self._reloadvgs(vgNames)) if vgName in vgNames]
```

Comment:
indentation does not match how we use list comprehensions. It should be:

    [vg for vgName, vg
     in six.iteritems(self._reloadvgs(vgNames)) if vgName in vgNames]

Or:

    [vg for vgName, vg
     in six.iteritems(self._reloadvgs(vgNames))
     if vgName in vgNames]

Or just simplify by removing the useless siz.iteritems now:

    [vg for vgName, vg in self._reloadvgs(vgNames).items()
     if vgName in vgNames]

And the code will be more clear if we separte the call that can fail:

    try:
        vgs = self._reloadvgs(vgNames)
    except se.VolumeGroupDoesNotExist as e:
        handle the error...

    return [vg for vgName, vg in vgs.items() if vgName in vgNames]

Reply:
`six` is removed in a different commit, but I will separate the call to reloadvgs.

The line split was done by `autopep8`. Actually, to me separating the for in two lines looks weird, but is good to know, I will take it into account from now on. Anyway, for things like this, it would be nice to have `coding guidelines` documented.

Reply:
We can add coding guidelines, not sure how easy it will be to get something that everyone agree with, but we can try.

Reply:
Lol, I know how difficult can it be to agree on something. Specially when some people have very strong opinions on some particular topics.

Personally, I do prefer to have coding guidelines, even if I don't like all of them. Helps to have the code more consistent, and not 'waste time' thinking on how to format the code. That is why I like having an auto formatter at hand.

But we can talk about it next week to see if it is worth or not. It is something more contributors have to agree on.